discovery/pkg/sync/endpoints_test.go

```diff
@@ -138,7 +141,166 @@ func TestUpdateEndpoints(t *testing.T) {
 		},
 	}
 	expectedPatch := `{"subsets":[{"addresses":[{"ip":"192.168.0.2"}],"ports":[{"port":8080}]},{"addresses":[{"ip":"192.168.0.3"}],"ports":[{"port":80}]}]}`
-	err := updateEndpoints(client, &newEndpoints, localmetrics.NewMetrics(), "testnode")
+	err := updateEndpoints(client, &newEndpoints)
 	require.NoError(t, err)
 	assert.Equal(t, expectedPatch, string(gotPatchBytes))
 }
+
+func TestDiscovererEndpointsMetrics(t *testing.T) {
+	backendName := "backend"
+	backendType := "backtype"
+	tests := []struct {
+		name              string
+		actionKind        string
+		endpoints         v1.Endpoints
+		existingendpoints v1.Endpoints
+		expectErr         bool
+		expectedCount     float64
+		expectedTimestamp float64
+		expectedLabels    map[string]string
```

Comment:
Doesn't look like there are assertions on the labels

Reply:
No, I didn't write tests for those. They are simple pass-through values. But initially, I had though to implement those.

Reply:
Got it! Should we remove them from this struct?